dataland-frontend/src/components/forms/parts/elements/basic/ExtendedDataPointFormField.vue

```diff
@@ -3,8 +3,8 @@
     <slot />
     <div>
       <FormKit type="group" name="dataSource">
-        <div class="next-to-each-other">
-          <div class="flex-1">
+        <div class="grid align-content-end">
+          <div class="col-8">
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/732ff0d8-1cc1-4b6d-8ffb-429ece2a31cb)
It seems that the values for GWh/Mio€ are not correctly filled

Reply:
Dood point. Fixed

Reply:
![image](https://github.com/d-fine/Dataland/assets/104447806/6518f330-483e-41d5-bcdd-4f4b327cae52)
I think if the value is empty and quality is set to NA it should not be displayed, this looks rather strange

Reply:
changed to 
"No data provided"

Reply:
Is it possible to also check if the comment is non null and if it is non null display it?

Reply:
Looks good now